server/datastore/mysql/cron_stats.go

```diff
@@ -0,0 +1,67 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/jmoiron/sqlx"
+)
+
+func (ds *Datastore) GetLatestCronStats(ctx context.Context, name string) (*fleet.CronStats, error) {
+	stmt := `SELECT id, name, instance, created_at, updated_at, stats_type, status FROM cron_stats WHERE name = ? ORDER BY created_at DESC LIMIT 1`
```

Comment:
Based on this, I think you should add an index on `name, created_at`. Or maybe it's the other way around, give it a try and check the explain on each to see which one is the right one, not sure if the condition or the ordering has precedence here for planning.

Reply:
Also... should this actually be ordered by `updated_at` instead? Wouldn't that be the latest?

Reply:
Yep, we already have the `name, created_at` index :) I tried them both and the condition takes precedence.

Reply:
We're ordering by `created_at` because we're really interested in the job that most recently started. In theory they should be the same, but `updated_at` won't necessarily correspond to the most recently started job. I'm thinking here of ways that stats for one run might be updated after the run has completed and another run started before the stats are updated for the first run.

Reply:
Oh silly me, I see the index now... great! Sorry for the noise.